qiskit/circuit/library/standard_gates/xx_plus_yy.py

```diff
@@ -84,12 +84,12 @@ class XYGate(Gate):
     def __init__(
         self, theta: ParameterValueType, beta: ParameterValueType = 0, label: Optional[str] = None
     ):
-        """Create new XY gate."""
+        """Create new XX+YY gate."""
```

Comment:
While we're changing this gate, would you mind maybe improving the docstring that was here before by documenting the arguments? 🙂

Reply:
Done.